packages/tanstack-start/src/utils/index.ts

```diff
@@ -0,0 +1,9 @@
+import { buildErrorThrower } from '@clerk/shared';
+
+export const isClient = typeof window !== 'undefined';
```

Comment:
This looks fine as you rely on vite to instantiate the module once for the client and once for the server. Just curious - is there any other reason you prefer constants instead of functions here?

Reply:
No reason, you are right this should be a function!